nix/mobile/ios/default.nix

```diff
@@ -27,23 +27,19 @@ let
         };
     };
 
-  selectedSources = [ status-go ];
-
-  # TARGETS
-  shell = mkShell {
-    buildInputs = [ xcodeWrapper ];
-  };
-
-in {
-  inherit xcodeWrapper;
+  selectedSources = [ fastlane status-go ];
 
   buildInputs = unique ([
+    cocoapods
+    fastlane.drv
```

Comment:
Can't say I like including `fastlane` here when it should already be taken care of by sitting on `selectedSources`. One way around this would be to have `fastlane` inherit `buildInputs` from `fastlane.drv` so as to expose that here implicitly, but that's not perfect either.

Reply:
But inheriting `buildInputs` from `fastlane.drv` doesn't add `fastlane` to the shell.